WIP - Sustainability dashboard documentation (#6141)
umbraco/UmbracoDocs: umbraco-cloud/getting-started/the-umbraco-cloud-portal/sustainability-dashboard.md

# Sustainability dashboard

The Sustainability Dashboard is designed to help users monitor and improve the environmental impact of their websites on Umbraco Cloud. The dashboard provides insights and metrics related to carbon footprint and sustainable practices, enabling organizations to align their digital presence with their sustainability goals.

## Key Features

- **Daily CO2 emission calculation**: The dashboard is updated daily with new CO2 emission estimates.
- **Historical data**: The dashboard tracks monthly and yearly CO2 emission estimates, allowing for trend analysis over time
- **Comparative analysis**: Users can easily compare CO2 emissions across different projects to identify high-impact areas and improvement opportunities.

## CO2 emission calculation methodology

> To estimate CO2 emissions from the infrastructure deployed to host Umbraco Cloud websites, we use [Cloud Carbon Footprint (CCF)](https://www.cloudcarbonfootprint.org/) followed by division of CO2 emission among websites according to their usage of shared resources.

Reviewer: ```suggestion
To estimate CO2 emissions from the infrastructure deployed to host Umbraco Cloud websites, we use [Cloud Carbon Footprint (CCF).](https://www.cloudcarbonfootprint.org/) followed by dividing CO2 emissions among websites according to their shared resource usage.
```

In order to improve the estimate for websites that are running in shared pools we divide emissions based on metrics/usage coefficients.

In its current iteration the sustainability dashboard shows CO2 emissions emitted by backend compute infrastructure - Azure App Service.

The summarized algorithm currently in use is:

```math
Project CO2 emissions = Sum of environment CO2 emissions
Environment CO2 emissions = Total CO2 emissions of backend compute infrastructure * Usage coefficient of the environemnts compute resources
```

### Cloud Carbon Footprint

> Cloud Carbon Footprint (CCF) provides a comprehensive methodology for estimating CO2 emissions in their [documentation](https://www.cloudcarbonfootprint.org/docs/methodology/). We use CCF in order to calculate the Sum of environment CO2 emissions. 

Reviewer: ```suggestion
Cloud Carbon Footprint(CCF) provides a comprehensive methodology for estimating CO2 emissions in their [documentation](https://www.cloudcarbonfootprint.org/docs/methodology/). We use CCF in order to calculate the Sum of environment CO2 emissions. 
```

```math
CCF CO2 emissions = Operational emissions + Embodied emissions
```

```math
Operational emissions = (Cloud provider service usage) x (Cloud energy conversion factors [kWh]) x (Cloud provider Power Usage Effectiveness (PUE)) x (grid emissions factors [metric tons CO2e])
Embodied emissions = Estimated metric tons CO2 emissions from the manufacturing of the datacenter
```

### Umbraco Cloud Usage

For websites on shared infrastructure in Umbraco Cloud, we calculate a usage coefficient to improve the accuracy of CO2 emission estimates. This coefficient divides the CO2 emissions of the shared pool among the websites using it.

> The usage coefficient for a website is based on metrics such as CPU or memory usage, usage coefficient for a database is based on DTUs used etc.

Reviewer: ```suggestion
The usage coefficient for a website is based on metrics such as:
* CPU
* Memory usage

 The usage coefficient for a database is based on DTUs used etc.
```

```

### Cloud Carbon Footprint

Cloud Carbon Footprint(CCF) provides a comprehensive methodology for estimating CO2 emissions in their [documentation](https://www.cloudcarbonfootprint.org/docs/methodology/). We use CCF to calculate the Sum of environmental CO2 emissions. 

```math
CCF CO2 emissions = Operational emissions + Embodied emissions
```

```math
Operational emissions = (Cloud provider service usage) x (Cloud energy conversion factors [kWh]) x (Cloud provider Power Usage Effectiveness (PUE)) x (grid emissions factors [metric tons CO2e])
Embodied emissions = Estimated metric tons CO2 emissions from the manufacturing of the datacenter
```

### Umbraco Cloud Usage

For websites on shared infrastructure in Umbraco Cloud, we calculate a usage coefficient to improve the accuracy of CO2 emission estimates. This coefficient divides the CO2 emissions of the shared pool among the websites using it.

The usage coefficient for a website is based on metrics such as:
* CPU
* Memory usage

 The usage coefficient for a database is based on DTUs used etc.

```math

> Environemnt CO2 emissions = Usage coefficient * CCF emissions

Author: ```suggestion
Website CO2 emissions = Usage coefficient * CCF CO2 emissions
```
